### PR TITLE
RFE: enable Travis CI builds on s390x platform

### PR DESCRIPTION
**Pull request by AnitaNayak**
building libseccomp on s390x

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,15 @@ language: c
 compiler:
   - gcc
 
+matrix:
+  include: 
+  - name: default
```
Comment (drakenclimber) [inline .travis.yml]: I would be tempted to name it x86_64, but I don't have a strong preference either way.

Comment (AnitaNayak) [reply .travis.yml]: Thank you. Updated .travis.yaml. s390x build is failing https://travis-ci.org/seccomp/libseccomp/jobs/618689980?utm_medium=notification. `tr: write error: Broken pipe`. Could you please give any pointer on this.

Comment (drakenclimber) [reply .travis.yml]: Hmmm... is it a Travis CI problem, a test problem on s390x problem, or something else entirely?

Do you have access to an s390x box?  I don't :(.  If so, do the tests run successfully on that box when you invoke them by hand?

Comment (pcmoore) [reply .travis.yml]: If it helps, I saw similar problems when I tried enabling the aarch64 builds.  Unfortunately I didn't have the time to debug the problem further.  While I haven't used it, Travis does have some provisions for debugging failed builds: https://docs.travis-ci.com/user/running-build-in-debug-mode

**Comment by pcmoore** [comment]
Hi @AnitaNayak, have you had a chance to look into the s390x failures on Travis CI?

**Comment by pcmoore** [comment]
While it would be really nice to add CI support for s390x, we haven't had an update from @AnitaNayak  in some time so I'm going to go ahead and close this PR.  If someone wants to finish this work, please let us know.

**Pull request closed without merging**